CHANGELOG.md

```diff
@@ -5,6 +5,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.1.1] - 2019-01-22
+
+### Changed
+- Update charts.js and embed.chart.frame.js to use Wazimap releases features.
+- Update settings_js.html to include SITE_TAGLINE & SITE-DESCRIPTION
+- Bandage those dists without metadata not to through: Uncaught TypeError: Cannot read property 'qualifier' of undefined
```

Comment:
I think this is cool for code comment but it should be worded differently for CHANGELOG... Something like _Ensure `dist` metadata are initialized in JS before being accessed to avoid `TypeError` on reading `undefined` objects._... or something similar